cypress/e2e/swap.test.ts

```diff
@@ -11,34 +11,31 @@ describe('Swap', () => {
   })
 
   it('can enter an amount into input', () => {
-    cy.get('#swap-currency-input .token-amount-input')
-      .clear()
-      .type('0.001', { delay: 200 })
-      .should('have.value', '0.001')
+    cy.get('#swap-currency-input .token-amount-input').clear().type('0.001').should('have.value', '0.001')
```

Comment:
in a separate pr i will move all these selectors to `data-cy` rather than using ids and classnames